cmd/io4edge-cli/cmd/root.go

```diff
@@ -44,6 +45,7 @@ func Execute() {
 }
 
 func init() {
-	rootCmd.PersistentFlags().StringVarP(&device, "device", "d", "", "IP address of io4edge devices with port e.g. 192.168.7.1:9999")
+	rootCmd.PersistentFlags().StringVarP(&device, "device", "d", "", "Instance name of device")
```

Comment:
I would strongly recommend that the user can specify instance,service,protocol with a single command line option, not with separate options. Separate options make only sense when parts are optional